docs/_data/sidebar.yml

```diff
@@ -0,0 +1,29 @@
+- title: Getting Started
+  children:
```

Comment:
I think you need to remove this sample `children` section.

Reply:
WIP, it has real pages now.